src/sections/Projects/Sistent/components/badge/code-block.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
```

Comment:
@subh05sus a extra block component is not required. there is already one reuse that and delete this file please.